### DISCUSSION TITLE
How to change Telegram Api parameters?

### DISCUSSION THREAD

**Discussion by TheRedM** [AzuraCast/AzuraCast#4414] [Q&A] [answered]
I wanted to add the ```disable_web_page_preview```  Telegram api parameter in Telegram webhook i don't know how to do that.

**Comment by BusterNeece** [comment, marked as answer]
@TheRedM This would likely be a new feature request, though it doesn't sound like it would be too difficult for us to add in. You should suggest it over at https://features.azuracast.com/
